File: Backend/plated/src/main/java/com/priyanshparekh/plated/recipe/dto/AddRecipeResponse.java
package com.priyanshparekh.plated.recipe.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class AddRecipeResponse {

    private Long id;
    private String name;
    private String description;
    private String cuisine;
    private String category;
    private int servingSize;
    private float cookingTime;

}

File: Frontend/Plated/core-model/src/main/java/com/priyanshparekh/core/model/recipe/AddRecipeResponse.kt
package com.priyanshparekh.core.model.recipe

data class AddRecipeResponse(
    val id: Long,
    val name: String,
    val description: String,
    val cuisine: String,
    val category: String,
    val servingSize: Int,
    val cookingTime: Float,
)
